static/app/views/discover/table/quickContext/styles.tsx

```diff
@@ -1,11 +1,12 @@
 import styled from '@emotion/styled';
 
+import type {FlexProps} from 'sentry/components/core/layout';
+import {Flex, Stack} from 'sentry/components/core/layout';
 import {space} from 'sentry/styles/space';
 
-export const ContextContainer = styled('div')`
-  display: flex;
-  flex-direction: column;
-`;
+export function ContextContainer(props: FlexProps) {
+  return <Stack {...props} />;
+}
```

Comment:
These exports should almost certainly be inlined instead? But fine to punt for now...

Reply:
Agree - there seems to be some type of Context namespace with ContextBody, ContextTitle etc here though, so I'm keeping as is. I'm happy if we drop the styled components

Reply:
all usages use additional `styled(ContextContainer)` 🙈